src/test/java/org/example/AppTest.java

```diff
@@ -31,4 +36,16 @@ public static Test suite() {
     public void testApp() {
         assertTrue( true );
     }
+
+    /**
+     * Test sorting method - sortedList
+     */
+    public void testList(){
+        String[] data = {"one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"};
+        String[] ex = {"eight", "five", "four", "nine", "one", "seven", "six", "ten", "three", "two"};
+        ArrayList<String> expected = (ArrayList<String>) Arrays.stream(ex).collect(Collectors.toList());
```

Comment:
Why do you need this transformation, probably it's better to create expected as a list? E.g. `Arrays.asList(...)` or `List.of(...)`